node/cmd/guardiand/query_test.go

```diff
@@ -410,80 +407,86 @@ func TestInvalidQueries(t *testing.T) {
 	signedQueryRequest, _ = createSignedQueryRequestForTesting(md.sk, perChainQueries)
 	signedQueryRequest.Signature[0] += 1 // Corrupt the signature.
 	md.signedQueryReqWriteC <- signedQueryRequest
 	require.Nil(t, md.waitForResponse())
 
 	// Query for an unsupported chain should fail. The supported chains are defined in supportedChains in query.go
 	md.resetState()
-	perChainQueries = []*gossipv1.PerChainQueryRequest{createPerChainQueryForTesting(vaa.ChainIDAlgorand, "0x28d9630", 2)}
-	signedQueryRequest, _ = createSignedQueryRequestForTesting(md.sk, perChainQueries)
-	md.signedQueryReqWriteC <- signedQueryRequest
-	require.Nil(t, md.waitForResponse())
-
-	// Query with no per-chain queries should fail.
-	md.resetState()
-	signedQueryRequest, _ = createSignedQueryRequestForTesting(md.sk, []*gossipv1.PerChainQueryRequest{})
-	md.signedQueryReqWriteC <- signedQueryRequest
-	require.Nil(t, md.waitForResponse())
-
-	// Query for an invalid chain should fail.
-	md.resetState()
-	perChainQueries = []*gossipv1.PerChainQueryRequest{createPerChainQueryForTesting(vaa.ChainIDPolygon, "0x28d9630", 2)}
-	perChainQueries[0].ChainId = uint32(math.MaxUint16) + 1 // Corrupt the chain ID.
+	perChainQueries = []*common.PerChainQueryRequest{createPerChainQueryForTesting(vaa.ChainIDAlgorand, "0x28d9630", 2)}
 	signedQueryRequest, _ = createSignedQueryRequestForTesting(md.sk, perChainQueries)
 	md.signedQueryReqWriteC <- signedQueryRequest
 	require.Nil(t, md.waitForResponse())
 
 	// Query for a chain that supports queries but that is not in the watcher channel map should fail.
 	md.resetState()
-	perChainQueries = []*gossipv1.PerChainQueryRequest{createPerChainQueryForTesting(vaa.ChainIDSepolia, "0x28d9630", 2)}
+	perChainQueries = []*common.PerChainQueryRequest{createPerChainQueryForTesting(vaa.ChainIDSepolia, "0x28d9630", 2)}
 	signedQueryRequest, _ = createSignedQueryRequestForTesting(md.sk, perChainQueries)
 	md.signedQueryReqWriteC <- signedQueryRequest
 	require.Nil(t, md.waitForResponse())
 
-	// Query for "latest" should fail.
-	md.resetState()
-	perChainQueries = []*gossipv1.PerChainQueryRequest{createPerChainQueryForTesting(vaa.ChainIDPolygon, "0x28d9630", 2)}
-	switch req := perChainQueries[0].Message.(type) {
-	case *gossipv1.PerChainQueryRequest_EthCallQueryRequest:
-		req.EthCallQueryRequest.Block = "latest"
-	}
-	signedQueryRequest, _ = createSignedQueryRequestForTesting(md.sk, perChainQueries)
-	md.signedQueryReqWriteC <- signedQueryRequest
-	require.Nil(t, md.waitForResponse())
-
-	// A per-chain query with no call data should fail.
-	md.resetState()
-	perChainQueries = []*gossipv1.PerChainQueryRequest{createPerChainQueryForTesting(vaa.ChainIDPolygon, "0x28d9630", 0)}
-	signedQueryRequest, _ = createSignedQueryRequestForTesting(md.sk, perChainQueries)
-	md.signedQueryReqWriteC <- signedQueryRequest
-	require.Nil(t, md.waitForResponse())
+	/*
+		 TODO: These tests should be done in common/query_test.go. Make sure they are there.
```

Comment:
The directive to make sure the tests are there is for future you and not the code inspectors, correct?

Reply:
Correct. The tests have been updated and the TODO removed.